doc/whats_new/v1.1.rst

```diff
@@ -117,6 +117,13 @@ Changelog
   (which behaves like `'arbitrary-variance'`) to `'unit-variance'` in version 1.3.
   :pr:`19490` by :user:`Facundo Ferrin <fferrin>` and :user:`Julien Jerphanion <jjerphan>`
 
+:mod:`sklearn.ensemble`
+.......................
+- |Enhancement| :class:`ensemble.RandomTreesEmbedding` now has a more informative
```

Comment:
There was no `get_feature_names_out` for that class previously, therefore let's drop the "more" to avoid any confusion.

```suggestion
- |Enhancement| :class:`ensemble.RandomTreesEmbedding` now has an informative
```